docs/reference/hacl/dh/ec/p256.md

```diff
@@ -0,0 +1,10 @@
+# P-256
+
+## API Reference
+
+```{doxygenfunction} Hacl_P256_dh_initiator
```

Comment:
This has doc comments you want to take?

Reply:
I think so? https://github.com/cryspen/hacl-packages/blob/main/include/Hacl_P256.h#L353-L364